input/pagecontent/reference.md

```diff
@@ -786,7 +788,7 @@ The following keywords (case-sensitive) are defined:
 
 In the above, `name` refers to a valid [item name](#item-names) and `id` to an [item identifier](#item-identifiers).
 
-Depending on the type of item being defined, keywords may be required, suggested, optional, or prohibited. The following table shows the relationship between declarations and keywords:
+Depending on the type of item being defined, keywords may be REQUIRED, suggested (SHOULD be used), OPTIONAL, or prohibited (MUST NOT be used). The following table shows the relationship between declarations and keywords:
```

Comment:
I initially used `RECOMMENDED`, but then I saw that in the key on line 810, it already used `SHOULD be used` -- so then I considered if I should just adopt the existing language from the key. The clincher for me was that `SHOULD` starts with the letter `S`, so it just reinforces the symbol we use in the table (as opposed to `RECOMMENDED`, which adds confusion to the symbols we use in the table).

Reply:
That's a good point. I guess I was thinking of the confusion between the table and the other sections for each time, but since `SHOULD` and `RECOMMENDED` are synonyms, it shouldn't really cause any issue.